docs/tutorials/istio.md

```diff
@@ -0,0 +1,191 @@
+# Configuring ExternalDNS to use the Istio Gateway Source
+This tutorial describes how to configure ExternalDNS to use the Istio Gateway source.
+It is meant to supplement the other provider-specific setup tutorials.
+
+**Note:** Using the Istio Gateway source requires Istio >=1.0.0.
+
+### Manifest (for clusters without RBAC enabled)
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: external-dns
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      containers:
+      - name: external-dns
+        image: registry.opensource.zalan.do/teapot/external-dns:latest
+        args:
+        - --source=service
+        - --source=ingress
+        - --source=istio-gateway
+        - --istio-namespace=custom-istio-namespace # omit to use the default (istio-system)
```

Comment:
This needs to be changed to the new flag `--istio-ingress-gateway`

Reply:
Very good eye.